### DISCUSSION TITLE
Wront font used for code display in Firefox

### DISCUSSION THREAD

**Discussion by greimers** [community/community#23421] [New to GitHub]
<p>Since yesterday my Firefox uses the wrong font (SF Mono Heavy) to render code on GitHub for example in the PR review.<br>
The text is rendered way too bold. (See screenshot below)</p>
<p>I have no idea why this is happening. I did not install a System or Firefox update nor can I remember making any other changes to the configuration. Also it works fine in Safari and Firefox displays all other websites correctly.</p>
<p>I am on macOS 11.4 with Firefox 89.<br>
Does anybody have an idea what might be causing this issue?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/1/1/1174df77b573cb93c6ff1ee46ca89c43633d8879.png" data-download-href="/uploads/short-url/2uqvZN2RXtM9VvzuIug5XjYqQ3v.png?dl=1" title="Screenshot" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/5694277/181076854-ae088249-bf8a-4ff0-8568-698bc2d4f407.png" alt="Screenshot" data-base62-sha1="2uqvZN2RXtM9VvzuIug5XjYqQ3v" width="690" height="341" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/1/1/1174df77b573cb93c6ff1ee46ca89c43633d8879_2_690x341.png, https://aws1.discourse-cdn.com/github/optimized/3X/1/1/1174df77b573cb93c6ff1ee46ca89c43633d8879_2_1035x511.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/1/1/1174df77b573cb93c6ff1ee46ca89c43633d8879_2_1380x682.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/1/1/1174df77b573cb93c6ff1ee46ca89c43633d8879_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot</span><span class="informations">2706×1340 510 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by greimers** [comment]
<p>I could finally resolve this by removing (disabling) the font “SF Mono” in macOS’ FontBook app.<br>
I would think this is a bug in Firefox to select the wrong font variation…</p>

**Comment by wlonkly** [comment]
<p>Appreciate you tracking this down. I took the liberty of <a href="https://bugzilla.mozilla.org/show_bug.cgi?id=1717255" rel="noopener nofollow ugc">opening a Firefox bug report</a> for this.</p>

**Comment by wlonkly** [comment, marked as answer]
<p>And from that bug report: I had an old version of SF Mono (pre-Catalina, I think) on my computer. Downloading the latest version <a href="https://developer.apple.com/fonts/" rel="noopener nofollow ugc">from Apple here</a> and the problem has gone away.</p>
<p><a class="mention" href="/greimers">@greimers</a>, can you check your font version/date in the inspector panel in Font Book?</p>

**Comment by greimers** [comment]
<p>Thanks, <a class="mention" href="/wlonkly">@wlonkly</a>, for opening the Firefox bug!<br>
For me it was the same as for you. I think I had installed the SF font face shortly after it was released. So I had it as a user-installed font.<br>
Re-Installing the latest version and removing the old user font solves the issue. (It was version 12.0d2e3 whereas the latest version is 16.0d2e1).</p>
